cmd/crd-puller/pull-crds.go

```diff
@@ -46,6 +45,7 @@ func main() {
 		Example: "",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			kubeconfigPath := cmd.Flag("kubeconfig").Value.String()
+			resourcesToSync := viper.GetStringSlice("resources-to-sync")
 			config, err := clientcmd.BuildConfigFromFlags("", kubeconfigPath)
```

Comment:
Let's adjust this to be more consistent with how we've evolved client construction since this code was written:

```go
loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
loadingRules.ExplicitPath = kubeconfigPath

startingConfig, err := loadingRules.GetStartingConfig()
if err != nil {
	return err
}

config, err := clientcmd.NewDefaultClientConfig(*startingConfig, nil).ClientConfig()
```

Reply:
Done.